File: ABC/ABC098/B.py
n = int(input())
s = str(input())

ans = 0
for i in range(1, n):
    f = s[:i]
    b = s[i:]
    cnt = 0
    for c in [chr(j + ord('a')) for j in range(26)]:
        if c in f and c in b:
            cnt += 1
    ans = max(ans, cnt)

print(ans)

File: ABC/ABC098/B.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    int N;
    string S;
    cin >> N >> S;

    int ans = 0;
    for (int i = 1; i < N; ++i) {
        bool used[2][26];
        fill(used[0], used[1] + 26, false);

        for (int j = 0; j < i; ++j) {
            used[0][S[j] - 'a'] = true;
        }
        for (int j = i; j < N; ++j) {
            used[1][S[j] - 'a'] = true;
        }

        int cnt = 0;
        for (int k = 0; k < 26; ++k) {
            cnt += (used[0][k] && used[1][k]);
        }

        ans = max(ans, cnt);
    }

    cout << ans << endl;
    return 0;
}
